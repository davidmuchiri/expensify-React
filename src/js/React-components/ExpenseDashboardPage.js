import React from 'react';
import ExpenseList from './expenseList';
import ExpenseListFilters from './expenseListFilters';
import ExpensesSummary from './ExpensesSummary';

const ExpenseDashboardPage = () => (
	<div>
		<ExpensesSummary />
		<ExpenseListFilters />
		<ExpenseList />
	</div>
);

export default ExpenseDashboardPage;
