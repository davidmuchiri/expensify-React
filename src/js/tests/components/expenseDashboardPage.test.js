import React, { Component } from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboardPage from '../../React-components/ExpenseDashboardPage';

test('should render Expense Dashboard', () => {
	const wrapper = shallow(<ExpenseDashboardPage />);
	expect(wrapper).toMatchSnapshot();
});
