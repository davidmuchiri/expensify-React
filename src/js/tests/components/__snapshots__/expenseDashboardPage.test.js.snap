// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Expense Dashboard 1`] = `
<div>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
